setup.py

```diff
@@ -11,16 +11,15 @@
     description='Data scrolling in the terminal',
     long_description=long_description,
     long_description_content_type='text/markdown',
-    author='Ben Ogorek',
-    author_email='baogorek@gmail.com',
+    author='Ben Ogorek, John Merfeld, Kevin Merfeld',
```

Comment:
Gonna pop in and change this to "John C. Merfeld," just to be consistent with my other stuff. But thanks a lot for doing the reformatting. I'll be sure to check in future; helps now that we have the linter hooked up to the PRs!